components/brave_wallet_ui/components/desktop/views/nfts/components/nfts.tsx

```diff
@@ -87,6 +93,21 @@ export const Nfts = (props: Props) => {
     setShowAddNftModal(value => !value)
   }, [])
 
+  const hideNftDiscoveryModal = React.useCallback(() => {
+    setShowNftDiscoveryModal(current => {
+      localStorage.setItem(LOCAL_STORAGE_KEYS.IS_ENABLE_NFT_AUTO_DISCOVERY_MODAL_HIDDEN, 'true')
+      return !current
+    })
+  }, [])
+
+  const onConfirmNftAutoDiscovery = React.useCallback(async () => {
+    await setNftDiscoveryEnabled(true)
+    hideNftDiscoveryModal()
+  }, [])
```

Comment:
Missing `hideNftDiscoveryModal` dependency

Reply:
Thank you. Added `hideNftDiscoveryModal` to dep list